Privacy enabled Proof of Authority configs

marlowl opened the pull request:
- Added a new docker-compose that runs a privacy enabled PoA chain
- Added `run-privacy-poa.sh` , this is just a small adjustment to `run.sh` but is necessary for the chain to produce blocks
- Made some name changes to avoid name confusion in the docker-compose and shell scripts
- Added a node to the prometheus config. Chain stops producing blocks when I tried keys from the other sample nodes.

claassistantio commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/PegaSysEng/besu-quickstart?pullRequest=48) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/PegaSysEng/besu-quickstart?pullRequest=48) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/PegaSysEng/besu-quickstart?pullRequest=48) it.</sub>

marlowl commented:
-  Switched back to `docker-compose_privacy.yml` and ` docker-compose_privacy_poa.yml`
- Updated `list.sh` to an if/elif/else structure with ` docker-compose_privacy_poa.yml`
- Removed `run-privacy-poa.sh` and updated `run-privacy.sh`. Tested it and `run-privacy.sh` starts the (normal) PoW chain.  `run-privacy.sh -c ibft2` starts the PoA privacy enabled chain

Also added `${QUICKSTART_POA_API-ibft}` to the `--rpc-http-api` flag. So the `run-privacy.sh` can list the validator addresses for the PoA privacy enabled chain aswell!

Thanks for the quick response @joshuafernandes!
